app/views/main_views.py

```diff
@@ -21,10 +21,18 @@
 Customer-specific configuration is required for these fields:
 '''
 base64_encoded_username_and_password = '<add your base64-encoded username:password here>'
-account_id = '5006144'
-messaging_application_id = '75b74d68-f200-4bf9-bdc5-e8a11bbef39e'
-from_phone_num = '+19198675703'
-to_phone_num = '+19198895989'
+headers = {'Authorization': 'Basic ' + base64_encoded_username_and_password}
+
+'''
+URL parameters
+'''
+account_id = '<Your Bandwidth Account Number>'
+
+'''
+POST body parameters
+'''
+messaging_application_id = '<Your Bandwidth Messaging Application ID>'
+from_phone_num = '<Your Source Telephone Number>'
```

Comment:
Not sure if mentioned elsewhere, would be nice to call out it's an E164 format number